editor.planx.uk/src/pages/FlowEditor/components/Team/components/MembersTable.tsx

```diff
@@ -7,16 +9,30 @@ import TableHead from "@mui/material/TableHead";
 import TableRow from "@mui/material/TableRow";
 import { AddButton } from "pages/Team";
 import React, { useState } from "react";
+import Permission from "ui/editor/Permission";
 
 import { StyledAvatar, StyledTableRow } from "./../styles";
-import { MembersTableProps } from "./../types";
-import { AddNewEditorModal } from "./AddNewEditorModal";
+import { MembersTableProps, TeamMember } from "./../types";
+import { EditorUpsertModal } from "./EditorUpsertModal";
+
+const TableButton = styled(Button)(({ theme }) => ({
```

Comment:
nit: I'd name this for what it's used for, i.e. something more like `EditUserButton`, given that it's only used for one purpose